src/main/target/SPRACINGH7ZERO/config.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "platform.h"
+
+#ifdef USE_TARGET_CONFIG
+
+#include "config_helper.h"
+
+#include "io/serial.h"
+#include "osd/osd.h"
+#include "pg/pg.h"
+
+static targetSerialPortFunction_t targetSerialPortFunction[] = {
+    { SERIAL_PORT_USART1, FUNCTION_MSP },
```

Comment:
Up to you as the target maintainer, but pre-configuring UARTs for MSP carries the risk that, whenever the user has connected a 'chatty' device sending data when powered up (RX, ESC, ...), the MSP processor will be overwhelmed by misformatted frames, and the flight controller will become unresponsive before the configuration can be changed.

Reply:
noted. this is OK for now, but prior to the 4.1 release I may well change the ports assigned to MSP by default based on user-feedback of the 3 H7 FC's with the 4.0 code.